admin/app/assets/javascripts/workarea/admin/modules/active_by_segment_tooltips.js

```diff
@@ -4,70 +4,70 @@
 WORKAREA.registerModule('activeBySegmentTooltips', (function () {
     'use strict';
 
-    var initTooltip = function (index, trigger) {
-            var $link = $(trigger),
-                $form = $link.closest('form');
+    var hiddenInput = JST['workarea/core/templates/hidden_input'],
 
-            $link.tooltipster(
-                _.assign({}, WORKAREA.config.tooltipster, {
-                    interactive: true,
-                    content: $($link.attr('href')),
-                    functionAfter: function(instance) {
-                        var $tooltip = instance.content();
-                        addHiddenInputs($form, $tooltip);
-                        updateMessages($link, $tooltip);
-                    }
-                })
-            );
+        getSelectedOptionText = function ($select) {
+            return $select.find('option:selected').map(function (_i, option) {
+                return $.trim($(option).text());
+            }).toArray();
         },
 
-        addHiddenInputs = function ($form, $tooltip) {
-            $form.remove('input[type=hidden]:data(activeBySegmentInput)');
+        updateMessages = function ($link, $content) {
+            var $message = $link.find('.active-field__segments-message'),
+                names = getSelectedOptionText($content.find('select')),
+                text = I18n.t(
+                    'workarea.admin.shared.active_field.by_segment',
+                    { count: names.length, names: names.join(', ') }
+                );
 
-            $tooltip.find('select').each(function (i, select) {
-                var $select = $(select),
-                    $input = null;
+            $message.html($('<strong />').text(text));
```

Comment:
The idea was to only use a `<strong>` if there were actually selected segments so it will pop out visually. If there are none, the "by segments" can fade into the background since most won't be using it most of the time. Can you update so the JS reflects that?